### PR TITLE
Docs: update comments regarding normalization factor for price and size

### PR DESCRIPTION
**Pull request by bigdefidev**

### CODE REVIEW

File: friktion/friktion/bid_details.py [hunk 1]
```diff
@@ -10,8 +10,8 @@
 class BidDetails:
     signer_wallet: PublicKey
     order_id: int
-    bid_size: int  # 1e9
-    bid_price: int  # 1e9
+    bid_size: int  # 1, the smart contract always initializes option token with 0 decimals
+    bid_price: int  # number of bid tokens per offer token. normalization factor of 1.
```
Comment (cassios) [inline friktion/friktion/bid_details.py:14]: not sure if I understood the comments. In the slack thread you said it should be 1e6

Comment (cassios) [reply friktion/friktion/bid_details.py:14]: answered on slack

**Comment by pdonorio** [comment]
Is this going to be merged @bigdefidev ?

**Comment by cassios** [comment]
I think it is obsolete. I will close the PR

**Pull request closed without merging**